src/components/profile/profile.tsx

```diff
@@ -67,13 +202,56 @@ const Profile = () => {
             .
           </p>
         </div>
-        <div style={{ margin: "1em 0" }}>
-          <Alert variant={user?.timeout ? "success" : "danger"}>
-            {user.timeout
-              ? "Your user session is active."
-              : "Your session has expired!"}
-          </Alert>
-        </div>
+        {userIVAs.length > 0
+          ? userIVAs.map((x, index) => {
+              return (
+                <Row key={x.id + index}>
+                  <Col xs={3}>
+                    {IVAType[x.type]}: {x.value}
```

Comment:
The type should be translated to a more readable description, like "Postal address" instead of "PostalAddress".

Reply:
Done 👍